package.json

```diff
@@ -9,7 +9,7 @@
   "dependencies": {
     "@edx/brand": "npm:@openedx/brand-openedx@1.2.2",
     "@edx/frontend-component-footer": "13.0.2",
-    "@edx/frontend-enterprise-catalog-search": "9.0.0",
+    "@edx/frontend-enterprise-catalog-search": "^10.3.0",
```

Comment:
nit: pin the dependency (similar to others; maintain convention) and re-run `npm install` to update package-lock.json.